src/features/metadata-instance-editor/Instance.js

```diff
@@ -524,18 +530,41 @@ class Instance extends React.PureComponent<Props, State> {
         return ops;
     }
 
+    /**
+     * Utility function to determine if instance should consider cascading policy
+     *
+     * @return {boolean} true if cascading policy is applicable
+     */
+    isCascadingPolicyApplicable = () => {
+        const { template, isFolderInstance }: Props = this.props;
+        const isTemplatedInstance = template.templateKey !== TEMPLATE_CUSTOM_PROPERTIES;
+        return isTemplatedInstance && isFolderInstance;
+    };
+
     /**
      * Utility function to determine if instance is editable
      *
      * @return {boolean} true if editable
      */
     canEdit(): boolean {
-        const { canEdit, onModification, onRemove, onSave }: Props = this.props;
-        return (
+        const {
+            canEdit,
+            onModification,
+            onRemove,
+            onSave,
+            template,
+            isCascadingPolicyApplicable,
+            isFolderInstance,
+        }: Props = this.props;
+        const isCustom = template.templateKey === TEMPLATE_CUSTOM_PROPERTIES;
+        const hasCascadePermission = isCustom || !isFolderInstance ? true : isCascadingPolicyApplicable;
+
+        return !!(
             canEdit &&
             typeof onRemove === 'function' &&
             typeof onSave === 'function' &&
-            typeof onModification === 'function'
+            typeof onModification === 'function' &&
+            hasCascadePermission
```

Comment:
the changes here looks like it could utilize the existing `canEdit` prop. e.g. in the `FolderMetadataModal` component, there is a map function to construct the `editors` prop that's passed to `MetadataInstanceEditor`

in this map function seems like a good opportunity to set the `canEdit` property based on the `template` and `isCascadingPolicyApplicable` values

otherwise the behavior could be confusing that setting a prop `canEdit = true` would still return false for this method because it's looking at several additional factors

Reply:
I went with your suggestion here and ended up utilizing the instance property for more than just canEdit. I'm also adding `isCascadingPolicyApplicable` and `isFolderInstance` here as well